examples/hello_world/finance/demo.py

```diff
@@ -0,0 +1,112 @@
+import time
+import random
+from collections import OrderedDict
+from maro.simulator import Env, DecisionMode
+from maro.simulator.scenarios.finance.common.common import MarketOrder, LimitOrder, StopOrder, StopLimitOrder, OrderMode, ActionType, DecisionEvent, CancelOrder, OrderDirection
+from maro.simulator.utils.common import tick_to_frame_index
+
+auto_event_mode = False
+start_tick = 10226  # 2019-01-01
+durations = 100
+max_ep = 1
+snapshot_resolution = 1
+
+env = Env(scenario="finance", topology="demo", start_tick=start_tick, durations=durations, decision_mode=DecisionMode.Joint, snapshot_resolution=snapshot_resolution)
+
+print(env.summary)
+for ep in range(max_ep):
+    metrics = None
+    decision_evts = None
+    is_done = False
+    actions = None
+
+    while not is_done:
+        metrics, decision_evts, is_done = env.step(actions)
```

Comment:
mv the None action step out of while

Reply:
put the non-None action step at the end of "while"

Reply:
ok